analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -601,10 +599,24 @@ def parse_with_plt_formatter(plist_file: str,
     return changed_files
 
 
-def parse_convert_reports(input_dirs: List[str],
-                          out_format: str,
-                          severity_map: Dict,
-                          trim_path_prefixes: List[str]) \
+class TrimPathPrefixHandler:
```

Comment:
If we want to keep this class lets move it to the common util module where the trim_path_prefixes function can be found, so other modules will be able to use this class too.